File: VB/SpreadsheetControl_WPF_API/SpreadsheetActions/ImportExportActions.vb
Imports System
Imports System.IO
Imports System.Diagnostics
Imports DevExpress.Spreadsheet

Namespace SpreadsheetControl_WPF_API

    Public Module ImportExportActions

        Public ExportToPdfAction As Action(Of IWorkbook) = AddressOf ExportToPdf

        Private Sub ExportToPdf(ByVal workbook As IWorkbook)
            workbook.Worksheets(0).Cells("D8").Value = "This document is exported to the PDF format."
'#Region "#ExportToPdf"
            Using pdfFileStream As FileStream = New FileStream("Documents\Document_PDF.pdf", FileMode.Create)
                workbook.ExportToPdf(pdfFileStream)
            End Using

'#End Region  ' #ExportToPdf
            Call Process.Start("Documents\Document_PDF.pdf")
        End Sub
    End Module
End Namespace

File: CS/SpreadsheetControl_WPF_API/SpreadsheetActions/ImportExportActions.cs
using System;
using System.IO;
using System.Diagnostics;
using DevExpress.Spreadsheet;

namespace SpreadsheetControl_WPF_API
{
    public static class ImportExportActions {

        public static Action<IWorkbook> ExportToPdfAction = ExportToPdf;

        static void ExportToPdf(IWorkbook workbook) {
            workbook.Worksheets[0].Cells["D8"].Value = "This document is exported to the PDF format.";

            #region #ExportToPdf
            using (FileStream pdfFileStream = new FileStream("Documents\\Document_PDF.pdf", FileMode.Create)) {
                workbook.ExportToPdf(pdfFileStream);
            }
            #endregion #ExportToPdf
            var p = new Process();
            p.StartInfo = new ProcessStartInfo(@"Documents\Document_PDF.pdf")
            {
                UseShellExecute = true
            };
            p.Start();
        }
    }
}
